components/Card/Achievement/index.vue

```diff
@@ -46,7 +70,31 @@
     </div>
 
     <div class="achievement__main-outro-nav">
-      <CardAchievementSwiper :data="contents" :is-show-pagination="true" />
+      <div class="achievement-nav">
+        <button slot="button-prev" class="swiper-prev-achievement">
+          <jds-icon
+            :class="{
+              'button-left' : true,
+              'button-left--disabled': currIndex === 0
+            }"
+            size="16px"
+            name="chevron-left"
+            @click="prevNav"
+          />
+        </button>
+        <div v-if="!isShowPagination" slot="pagination" class="swiper-pagination" />
```

Comment:
maybe we can switch `isShowPagination` default value to true, cause so it will be more readable with the logic..
cause pagination is appear in this section right? thankyouu @doohanas

Reply:
Yes, it appears when the width of the screen is less than 640. And the changes in [here ](https://github.com/jabardigitalservice/dedi-landing-page/pull/72/commits/fdaa8fd6b3a492dd402abc64d4e470d6e394abb3)kak @yoslie. 
And for the remove `components/Card/Achievement/Swiper.vue` in [here](https://github.com/jabardigitalservice/dedi-landing-page/pull/72/commits/d645fb3ce4a388e24e2fa8ed7c1447f95eb20f54)